src/domain/common/haittojenhallinta/CommonProcedureTips.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box } from '@chakra-ui/react';
+import { Accordion, IconDocument, IconLightbulb, Link } from 'hds-react';
+import React from 'react';
+
+import { Trans, useTranslation } from 'react-i18next';
+import { useLocalizedRoutes } from '../../../common/hooks/useLocalizedRoutes';
+import Text from '../../../common/components/text/Text';
+
+type CardLink = { number: number; heading: string; content: string; cardLinks: number[] };
+
+const CommonProcedureTips: React.FC = () => {
+  const { CARD } = useLocalizedRoutes();
+  const { t } = useTranslation();
+
+  const renderProcedureTips = () => {
+    const tipContents: CardLink[] = [];
+    for (let i = 1; i <= 14; i++) {
+      tipContents.push({
+        number: i,
+        heading: t(`hankeForm:haittojenHallintaForm:procedureTips:common:${i}:heading`),
+        content: t(`hankeForm:haittojenHallintaForm:procedureTips:common:${i}:content`),
+        cardLinks: [],
+      });
+    }
+    tipContents.find((tip) => tip.number === 1)?.cardLinks.push(8, 9);
+    tipContents.find((tip) => tip.number === 2)?.cardLinks.push(2, 3, 5, 6, 7);
+    tipContents.find((tip) => tip.number === 3)?.cardLinks.push(5, 7);
+    tipContents.find((tip) => tip.number === 4)?.cardLinks.push(10);
+    tipContents.find((tip) => tip.number === 9)?.cardLinks.push(1);
+
+    const procedureTips = tipContents.map((tip) => (
+      <Accordion
+        key={tip.number}
+        heading={tip.heading}
+        size="s"
+        headingLevel={5}
+        theme={{
+          '--content-font-size': 'var(--fontsize-body-m)',
+          '--header-font-size': 'var(--fontsize-heading-s)',
+        }}
+      >
+        <Box as="p" mb="var(--spacing-s)">
+          <Trans
+            i18nKey={`hankeForm:haittojenHallintaForm:procedureTips:common:${tip.number}:text`}
+            components={{
+              a: (
+                <a
+                  className="hds-link hds-link--medium"
+                  aria-label={t('common:components:link:openInNewTabAriaLabel')}
+                  target="_blank"
+                >
+                  External link
+                  <span
+                    className="hds-icon icon hds-icon--link-external hds-icon--size-s vertical-align-medium-icon"
+                    aria-hidden="true"
+                  ></span>
+                </a>
+              ),
+              span: <span />,
+            }}
+          >
+            replaced
+          </Trans>
+        </Box>
+        {tip.cardLinks.length > 0 && (
+          <Box as="p" mb="var(--spacing-s)">
+            <strong>
+              {t('hankeForm:haittojenHallintaForm:procedureTips:common:nuisanceCardLinks')}
+            </strong>
+            {tip.cardLinks.map((linkId) => (
+              <Link
```

Comment:
Any idea why opening a card link opens a new tab where there is no session (user is not logged in)?
![image](https://github.com/user-attachments/assets/dabb1fd9-5a39-4d0a-b3eb-b7035e3909fe)

Reply:
The same seems to happen everywhere in Haitaton, for example if you open Hankkeet yleisillä aluella from the navbar to a new window..

Reply:
The same seems to happen everywhere in Haitaton, for example if you open Hankkeet yleisillä aluella from the navbar to a new window..